build.sc

```diff
@@ -7,12 +7,12 @@ import com.goyeau.mill.scalafix.StyleModule
 import de.tobiasroeser.mill.integrationtest._
 import mill._
 import mill.scalalib._
-import mill.scalalib.api.Util.scalaNativeBinaryVersion
+import mill.scalalib.api.ZincWorkerUtil.scalaNativeBinaryVersion
```

Comment:
`Util` is deprecated

Reply:
https://github.com/com-lihaoyi/mill/blob/7f073d11df739c34b832eb6333500c65e26c63da/scalalib/api/src/mill/scalalib/api/ZincWorkerUtil.scala#L155